src/matrixTransforms/Round.cpp

```diff
@@ -12,10 +12,20 @@ bool Round::load() {
 
     double *val_data = valData();
     const uint32_t cap = capacity();
+    const uint32_t digits = (uint32_t)fit.global_params(0);
 
-    for (uint32_t i = 0; i < cap; i++) {
-        val_data[i] = std::nearbyint(val_data[i]);
+    if(digits == 0) {
+      for (uint32_t i = 0; i < cap; i++) {
+          val_data[i] = std::nearbyint(val_data[i]);
+      }
     }
+    else {
+      const double factor = pow(10.0, (double)digits);
+      for (uint32_t i = 0; i < cap; i++) {
+        val_data[i] = std::nearbyint(val_data[i] * factor) / factor;
+      }
```

Comment:
Since division much slower than multiplication, we can do a little optimization here of pre-computing `inv_factor = 1/factor` in a separate variable, then doing `std::nearbyint(val_data[i] * factor) * inv_factor`

[This reference](https://www.agner.org/optimize/instruction_tables.pdf) lists multiplication (FMUL) at 1 cycle, and division (FDIV) at 4-5 cycles for 10th gen Intel chips. This will slightly mess with the precision of our operation